scripts/scil_surface_convert.py

```diff
@@ -67,24 +63,19 @@ def main():
     assert_inputs_exist(parser, args.in_surface)
     assert_outputs_exist(parser, args, args.out_surface)
 
-    if args.xform:
-        with open(args.xform) as f:
-            content = f.readlines()
-        xform = [x.strip() for x in content]
-        xform_matrix = extract_xform(xform)
-        xform_translation = xform_matrix[0:3, 3]
-    else:
-        xform_translation = [0, 0, 0]
+    _, ext = os.path.splitext(args.in_surface)
+    if ext not in ['.vtk', '.vtp', '.fib', '.ply', '.stl', '.xml', '.obj']:
+        if args.reference is None:
+            parser.error('The reference image is required for FreeSurfer '
+                         'surfaces.')
```

Comment:
most of freesurfer surfaces don't have any of these extensions...(lh.white, pial, ....)
I try to understand.

Reply:
Exactly, Freesurfer have no extension so this if just check if the file has any of the known/supported extension, otherwise its very likely freesurfer